apps/bluetooth/app.js

```diff
@@ -112,14 +113,16 @@ module.exports = function (activity) {
     logger.debug(`speak: ${text}`)
     if (!textIsEmpty(text)) {
       return activity.setForeground().then(() => {
-        if (wifi.getWifiState() === wifi.WIFI_CONNECTED) {
-          return activity.tts.speak(text, { impatient: false }).catch((err) => {
-            logger.error('play tts error: ', err)
-          })
-        } else if (alternativeVoice != null) {
-          logger.debug('No wifi connection, play alternative voice.')
-          return activity.playSound(alternativeVoice)
-        }
+        networkAgent.getNetworkStatus().then((reply) => {
```

Comment:
IMO, the `getNetworkStatus()` shall only return `state`, so the code will like:
```suggestion
        networkAgent.getNetworkStatus().then((state) => {
```

Reply:
I'll refactor the mechanism of parsing result from the network service to make the return value easier to use.